### DISCUSSION TITLE
Mqtt/Json Abfrage um Wert in PHP auszuwerten

### DISCUSSION THREAD

**Discussion by steven1234561** [jomjol/AI-on-the-edge-device#638] [General]
Hallo, 

als erstes: ich liebe dieses Projekt!! 

Mein "Problem":
Ich Frage die Gas-Tank-Zähler mehrere Häuser ab. Aktuell über dyndns/portfreigabe/json - und speichere diese in unserer Software (PHP/Mysql) ab um damit den Tankbestand zur überwachen. Das funktioniert eigentlich wunderbar. 

Problem ist nur der offene Zugang auf den ESP32. Ich hatte schon gelesen das eine Authentifizierung nicht unbedingt das nächste Feature ist... Also wäre aktuell aus meiner Sicht nur der Umweg über MqTT möglich - oder übersehe ich eine andere Möglichkeit??

Am liebsten wäre mir, wenn der Json String einfach an eine Webadresse "geschickt" werden würde ähnlich dem Mqtt Verfahren z.b: https://demo.de/aiontheedge.php?topic=zaehler1&json=_JSON_STRING_

Ich bin aber für alle Ideen offen :-) 

Vielen Dank schon im Voraus

Steven

**Comment by jomjol** [comment]
Verschlüsselte Zugänge und Übertragung sehe ich eher für kommerzielle Anwendungen. Daher aktuell nicht auf der Roadmap dieses GitHubs.

**Reply by steven1234561** [reply]
Ok - Schade! - Ich dachte auch mehr an eine einfach htaccess passwort abfrage nicht gleich an eine komplette Sicherheitslösung.. Aber alles klar :-) Danke für die schnelle Antwort.
